packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -7109,3 +7127,13 @@ class SkipPaintingRenderObject extends RenderProxyBox {
   @override
   void paint(PaintingContext context, Offset offset) { }
 }
+
+class AccentColorTextEditingController extends TextEditingController {
```

Comment:
Nit: I think this should be private (`_AccentColorTextEditingController`) just to avoid any confusion.